### PR TITLE
[SD-1544] - UI: Clicking edit icon in list/grid is invoking preview befo...

### PR DESCRIPTION
**Pull request by sjunaid**
...re authoring

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: app/scripts/superdesk-archive/views/media-box-grid.html [hunk 1]
```diff
@@ -1,43 +1,45 @@
 <div class="archiving-progress" style="width:{{ _progress }}%" ng-if="_progress"></div>
 
-<div class="error-box" ng-show="item.archiveError">
-    <p class="message">There was an <b>error</b> archiving this item</p>
-    <div class="buttons">
+<div ng-click="clickAction(item)">
+    <div class="error-box" ng-show="item.archiveError">
+        <p class="message">There was an <b>error</b> archiving this item</p>
```
Comment (petrjasek) [inline app/scripts/superdesk-archive/views/media-box-grid.html]: missing `translate`

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**